packages/dev-console-standalone/src/hooks/useToast.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {createContext, useContext, useMemo} from 'react';
+import {Frame} from '@shopify/polaris';
+
+import {usePolarisToast} from './usePolarisToast';
+
+interface ToastContextProps {
+  show: ReturnType<typeof usePolarisToast>[1];
+}
+
+const ToastContext = createContext<ToastContextProps | undefined>(undefined);
+
+interface ToastProviderProps {
+  children: React.ReactNode;
+}
+
+const style = {
```

Comment:
I feel like with just one property, this could be done inline.

Reply:
I had no hand in writing this. We'll be replacing Polaris soon anyway, so this'll be rewritten if we end up using Toast